src/style.css

```diff
@@ -1,3 +1,79 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:500&display=swap';
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  --main-color: #000;
+  --blue-color: #00acdc;
+
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
+.container {
+  padding-left: 50px;
+  padding-right: 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```

Comment:
Don't fix container size (if there is no such a requirement). Fixed padding on the container can cause issues with responsiveness. Consider using relative units or media queries to handle different screen sizes.